### PR TITLE
Resume for Grading

### PR DESCRIPTION
**Pull request by hnbarr**
## Checkpoint Rubric
This is the rubric that your instructor will use to grade your checkpoints. Please do not edit. 

**Checkpoint 1**
- [ ] A CSS file is effectively linked to the HTML file in the Head section: 15 points
- [ ] A link to a CSS reset file is included in the Head section of the HTML file: 15 points 
- [ ] Appropriate use of Semantic Elements: 15 points
- [ ] Effective use of box model: 15 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Linter passing: 10 points
- [ ] The student demonstrated mastery of HTML/CSS by building a website of professional caliber: 15 points

**Checkpoint 2**
- [ ] Effective use of Flexbox layout: 15 points
- [ ] Responsive site (resizes to fit desktop, tablet, and mobile views): 15 points 
- [ ] Effective use of CSS Grid: 15 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Site is hosted on github: 15 points
- [ ] Linter passing: 10 points
- [ ] The student demonstrated mastery of HTML/CSS and Responsive Web Design by building a website of professional caliber: 15 points

**Checkpoint 3**
- [ ] Use of HTML: 15 points
- [ ] Use of CSS: 15 points
- [ ] Use of JQuery: 15 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Linter passing: 10 points
- [ ] The application has a professional look: 15 points
- [ ] The application functions as it should: 15 points

### CODE REVIEW

File: 01week/resume/index.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+<head>
+ <title>HB Resume</title>
+ <link rel="stylesheet" type="text/css" media="screen" href="./css/style.css">
+</head>
+<!-- command forward slash -->
+<body>
+    <!-- left side of webpage -->
+    <div class="row">
```
Comment (DinhH) [inline 01week/resume/index.html:10]: I would like to see a bit more use of semantic tags. Example, your class row can be <main> then the columns can be <section>.

File: 01week/resume/index.html [hunk 2]
```diff
@@ -6,6 +6,7 @@
 </head>
 <!-- command forward slash -->
 <body>
+    <!-- left side of webpage -->
     <div class="row">
```
Comment (DinhH) [inline 01week/resume/index.html:10]: Great work on the comments!

Comment (hnbarr) [reply 01week/resume/index.html:10]: Thank you!!

File: 01week/resume/index.html (hunk 1 above)
Comment (hnbarr) [reply 01week/resume/index.html]: Okay, thank you and should I add those into my page and re-commit as final.2?

Comment (hnbarr) [reply 01week/resume/index.html:10]: Perfect, that's what I'm going to do! I'll have it done tomorrow!

Comment (hnbarr) [reply 01week/resume/index.html:10]: Or is it too late? I see that this has been merged into gh-pages today.

**Pull request merged**